src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -58,12 +59,36 @@ public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext
             DocumentCollectionInfo documentCollectionLocation;
             DocumentCollectionInfo leaseCollectionLocation;
             ChangeFeedProcessorOptions processorOptions = BuildProcessorOptions(attribute);
+            
             processorOptions.StartFromBeginning = attribute.StartFromBeginning;
+
             if (attribute.MaxItemsPerInvocation > 0)
             {
                 processorOptions.MaxItemCount = attribute.MaxItemsPerInvocation;
             }
 
+            var startFromTime = DateTime.UtcNow;
+
+            if (!string.IsNullOrEmpty(attribute.StartFromTime) && 
+                !DateTime.TryParse(attribute.StartFromTime, out startFromTime))
```

Comment:
You can defer the variable declaration to the `out` definition":

```suggestion
                !DateTime.TryParse(attribute.StartFromTime, out DateTime startFromTime))
```